src/main/java/com/minecolonies/coremod/items/ItemSupplyChestDeployer.java

```diff
@@ -80,6 +78,17 @@
     private static final int OFFSET_Y = -2;
 
     /**
+     * Our guide Book.
+     */
+    private static final ResourceLocation GUIDE_BOOK = new ResourceLocation(Constants.MOD_ID + ":book/minecolonies.xml");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "GUIDE_BOOK" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)